terraform/modules/apps/content-store/variables.tf

```diff
@@ -40,3 +40,9 @@ variable "execution_role_arn" {
   description = "For use during bootstrapping"
   type        = string
 }
+
+variable "desired_count" {
+  description = "Desired count of Application instances"
+  type        = number
+  default     = 0
```

Comment:
Is this intended to default to zero?

Reply:
Originally when we put the `desired_count`, these were `0` but after the pairing session where I see you put it as default one we can change that.